helm/examples/efs/hpcc-efs/templates/hpcc-efs-server.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "efs-provisioner-%s" (include "hpcc-efs.fullname" $) }}
+data:
+  file.system.id: {{ .Values.efs.id }}
+  aws.region: {{ .Values.efs.region }}
+  provisioner.name: aws.io/aws-efs
+  dns.name: ""
```

Comment:
curious what this is for / if needed?

Reply:
This is required by EFS implementation I think